src/widget/wwaveformviewer.h

```diff
@@ -49,8 +49,6 @@ private slots:
         m_waveformWidget = nullptr;
     }
 
-    void slotWaveformUpdated();
```

Comment:
Was there an issue with keeping this here? If so, you'd need to move this to all of the renderers, not just GLSL.